source/fundamentals/connection/connect.txt

```diff
@@ -50,13 +51,15 @@ connect to a MongoDB deployment. It instructs the driver on how it should
 connect to MongoDB and how it should behave while connected. The following
 figure explains each part of a sample connection URI:
 
-.. figure:: /includes/figures/connection_uri_parts.png
-   :alt: Connection String parts figure
+.. figure:: /includes/figures/dns_seedlist_connection_string_parts.png
+   :alt: Each part of the connection string
 
-This figure uses the :manual:`Standard Connection String Format </reference/connection-string/#std-label-connections-standard-connection-string-format>`,
-``mongodb`` for the protocol. You can also use the :manual:`DNS Seed List Connection Format </reference/connection-string/#dns-seed-list-connection-format>`,
-``mongodb+srv``, if you want more flexibility of deployment and the ability
-to change the servers in rotation without reconfiguring clients.
+In this example, we connect to an Atlas MongoDB deployment that has a
```

Comment:
Style: Write to the user by using second person. ([ref](https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/))

```suggestion
In this example, you connect to an Atlas MongoDB deployment that has a
```